packages/bsp/common/usr/lib/armbian/armbian-hardware-optimization

```diff
@@ -78,15 +78,36 @@ prepare_board() {
 	grep -q ondemand /etc/default/cpufrequtils
 	if [ $? -eq 0 ]; then
 		echo ondemand >/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
-		cd /sys/devices/system/cpu
-		for i in cpufreq/ondemand cpu0/cpufreq/ondemand cpu4/cpufreq/ondemand ; do
-			if [ -d $i ]; then
-				echo 1 >${i}/io_is_busy
-				echo 25 >${i}/up_threshold
-				echo 10 >${i}/sampling_down_factor
-				echo 200000 >${i}/sampling_rate
+
+		case ${BOARDFAMILY} in
```

Comment:
This PR introduces a `case` without a matching `esac`. That's a syntax error for sure. @150balbes @igorpecovnik 
I've the fix ready (plus a `shfmt`) in #4430  but it's draft / untested.

Reply:
Oh. I expect code that is sent here is tested :( 

> but it's draft / untested.

Fixing untested code with another untested is usually not worse :)

Reply:
The code was checked, but when creating a PR, it was not completely copied into the created PR.  :(

Reply:
Hopefully this automated check https://github.com/armbian/build/pull/4431 will help finding this earlier. Will be enabled at PR.